bin/compile

```diff
@@ -30,7 +30,13 @@ curl --silent --retry 5 $BASE_URL/$SIGNATURE --output $SIGNATURE_PATH
 curl --silent --retry 5 $BASE_URL/$CHECKSUMS --output $CHECKSUMS_PATH
 
 echo "Verifying download" | indent
-gpg --quiet --keyserver pgp.mit.edu --recv 7BFB4EDA || gpg --quiet --keyserver keys.gnupg.net --recv 7BFB4EDA
+
+for i in {1..4}; do
```

Comment:
The `{1..4}` may be a bash-ism, so technically disallowed from `/bin/sh`.

Reply:
`/bin/sh` is a symlink to [`dash`] on Heroku, but you are correct, this is indeed a bash-ism (also works in zsh), but in dash, this doesn't loop:

```
$ cat buildpack-testing.sh
#!/bin/sh

for i in {1..4}; do
  echo $i
done

$ dash buildpack-testing.sh
{1..4}

$ bash buildpack-testing.sh
1
2
3
4
```

(the above from Ubuntu 16.04)

Reply:
Ok, cool.  Thanks for verifying that in dash.